projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -232,39 +251,31 @@ export default function SubscriptionControls( {
 					help={ () => {
 						if ( ! subscriberCount || subscriberCount < 1 ) {
 							return __(
-								'This will remain hidden on your website until you have at least one subscriber.',
+								'This will remain hidden until there is at least one subscriber.',
 								'jetpack'
 							);
 						}
 					} }
 				/>
-
 				<ToggleControl
 					label={ __( 'Place button on new line', 'jetpack' ) }
 					checked={ buttonOnNewLine }
 					onChange={ () => {
 						setAttributes( { buttonOnNewLine: ! buttonOnNewLine } );
 					} }
 				/>
-			</PanelBody>
-			{ ! isSimpleSite() && (
-				<PanelBody
-					title={ __( 'Success Message Text', 'jetpack' ) }
-					initialOpen={ false }
-					className="wp-block-jetpack-subscriptions__successpanel"
-				>
+				{ ! isSimpleSite() && (
 					<TextareaControl
 						value={ successMessage }
-						label={ __( 'Success Message Text', 'jetpack' ) }
-						hideLabelFromVision={ true }
+						label={ __( 'Success message', 'jetpack' ) }
 						help={ __(
-							'Save your custom message to display when a user subscribes your website.',
+							'Edit the message displayed when a user subscribes to the newsletter.',
```

Comment:
I personally find the use of "newsletter" here a bit confusing, since what Jetpack offers isn't strictly a newsletter. I think it makes sense to use the word as a keyword for the block, but I do not know if I would use the word in the UI.

Reply:
Gotta be careful not to unintentionally mix customers up with Newsletter form variation and Subscribe block — longer term these two blocks need to work more seamlessly together but meanwhile might be best to use terminology to differentiate?

<img width="682" alt="Screenshot 2022-02-09 at 14 58 52" src="https://user-images.githubusercontent.com/87168/153206350-3ccaf99c-b79b-4492-92a4-21bc0b2f00f0.png">
<img width="772" alt="Screenshot 2022-02-09 at 14 58 46" src="https://user-images.githubusercontent.com/87168/153206353-91a4a82f-f611-4cdb-86f1-9679cef68910.png">


<img width="287" alt="Screenshot 2022-02-09 at 14 59 21" src="https://user-images.githubusercontent.com/87168/153206341-57fe59cc-cdbd-4259-8d6d-03876f953e43.png">